src/Mpociot/Versionable/VersionableTrait.php

```diff
@@ -175,7 +175,11 @@ protected function versionablePostSave()
             $version->versionable_id   = $this->getKey();
             $version->versionable_type = get_class($this);
             $version->user_id          = $this->getAuthUserId();
+            
+            $versionedHiddenFields = isset($this->versionedHiddenFields) ? $this->versionedHiddenFields : [];
```

Comment:
Does it work if you change this to:

```php
$versionedHiddenFields = $this->versionedHiddenFields ?? [];
```

Reply:
Yes it does. I have pushed up a new commit to address that.